tests/jobs.py

```diff
@@ -95,7 +94,7 @@ def test_trigger_controller_dag(self):
         target_dag.clear()
 
         scheduler = SchedulerJob()
-        queue = mock.Mock()
```

Comment:
The `Mock` class was already being imported, so I cleaned up these constructor calls